nemo/collections/nlp/models/language_modeling/megatron_t5_prompt_learning_model.py

```diff
@@ -339,6 +339,14 @@ def inference_epoch_end(self, outputs):
         self.log('val_loss', averaged_loss, prog_bar=True, rank_zero_only=True)
         self.log('val_acc', val_acc, prog_bar=True, rank_zero_only=True)
 
+        logging.info(f'val_loss: {averaged_loss}')
+
+        # Save inference ready .nemo checkpoint version
+        if self.cfg.get("save_intermediate_nemo_file", True):
```

Comment:
could we make the flag consistent with PTL's `on_validation_end`? maybe `save_nemo_on_validation_end`

Reply:
yes